content/akeneo-web-api.yaml

```diff
@@ -2026,6 +2026,142 @@ paths:
         406:
           $ref: "#/responses/406Error"
 
+  /api/rest/v1/measure-family:
+    get:
+      summary: Get list of measure family
+      operationId: "measure_family_get_list"
+      description: This endpoint allows you to get a list of measure families. Measure families are paginated and sorted by code.
+      tags:
+        - Measure Family
+      responses:
+        200:
+          description: Return measure families paginated
+          schema:
+            title: Measure Families
+            type: object
+            allOf:
+              - $ref: '#/definitions/Pagination'
+              - properties:
+                  _embedded:
+                    type: object
+                    properties:
+                      _items:
+                        type: array
+                        items:
+                          $ref: '#/definitions/MeasureFamilyList'
+          examples: {
+            _links: {
+                "self": {
+                  "href": "https://demo.akeneo.com/api/rest/v1/measure-family?page=1&limit=19&with_count=false"
+                },
+                "first": {
+                  "href": "https://demo.akeneo.com/api/rest/v1/measure-family?page=1&limit=19&with_count=false"
+                }
+              },
+            "current_page": 1,
+            "items_count": 18,
```

Comment:
@NolwennP My bad, I told Olivier to do that, because it's added by default (no performance impact here because we are reading from a file). But actually it's maybe a bad idea because by default, `count` is deactivated...